reference-architecture/gcp/deployment-manager/core.jinja

```diff
@@ -72,12 +72,21 @@ resources:
             #cloud-config
             write_files:
             - content: |
-                DEVS='/dev/disk/by-id/google-docker'
-                VG='docker-vol'
-                DATA_SIZE='95%VG'
-                EXTRA_DOCKER_STORAGE_OPTIONS='--storage-opt dm.basesize=3G'
-              path: /etc/sysconfig/docker-storage-setup
+                DOCKER_STORAGE_OPTIONS='--storage-driver overlay2'
```

Comment:
Are you providing a device or due to the health checks are you just increasing /var ?